docs/python/doxygenlib/cdWriterDocstring.py

```diff
@@ -523,6 +523,18 @@ def __getPythonObjectAndPath(self, parentPath, overloads):
 
         (obj, pypath, jumped) = self.__getPythonObjectByPath(pypath)
 
+        if jumped and overloads[0].isFunction():
+            # We found a module-level function in our module through a permissive
+            # search (jumped=True).  Do some further vetting.
+            if obj and hasattr(obj, "__module__") and \
+                not parentPath[-1].name.startswith(obj.__module__.split(".")[1]):
+                # The doxygen object clearly indicates it's from another module.
+                obj = None
+            elif type(obj).__module__ != 'Boost.Python':
```

Comment:
I don't think the module name changed with `pxr_boost`, but seeing this reminds me that you didn't explicitly confirm if you've tested this with `pxr_boost` yet...